src/common/mfu_flist.h

```diff
@@ -169,27 +169,27 @@ void mfu_flist_free(mfu_flist* flist);
  * set directory permission bits in order to walk into a directory,
  * whose permission bits are otherwise restrictive (e.g., useful for recursive unlink) */
 void mfu_flist_walk_path(
-    const char* path,    /* IN  - path to be walked */
-    int use_stat,        /* IN  - whether to stat each item (1) or not (0) */
-    int dir_permissions, /* IN  - whether to set directory permission bits (1) or not (0) during walk */
-    mfu_flist flist      /* OUT - flist to insert walked items into */
+    const char* path,         /* IN  - path to be walked */
+    int use_stat,             /* IN  - whether to stat each item (1) or not (0) */
```

Comment:
While we're at it, I think it'd be nice to roll this "use_stat" flag into the walk options structure as well.

Reply:
@adammoody use stat by default?

Reply:
Good question, let's turn it on by default.